tailwind.config.js

```diff
@@ -61,51 +61,64 @@ module.exports = {
+    require(dir + "/components/layout/grid-gap.js")(),
+    require(dir + "/components/logo/logo.js")(),
+    require(dir + "/components/media/embed-container.js")(),
+    require(dir + "/components/responsive-spacing/responsive-spacing.js")(),
+    require(dir + "/components/skiplink/skiplink.js")(),
+    require(dir + "/components/tables/borderless.js")(),
+    require(dir + "/components/typography/modular-typography.js")(),
+    require(dir + "/components/typography/styles.js")(),
+    require(dir + "/components/typography/wysiwyg.js")(),
+    require(dir + "/components/shadow/text-shadow.js")(),
 
     // @tailwind utilities;
-    require(dir + '/utilities/accessibility/accessibility-hidden.js')(),
-    require(dir + '/utilities/accessibility/sr-only-text.js')(),
-    require(dir + '/utilities/link/link.js')(),
-    require(dir + '/utilities/link/link-fontweight.js')(),
-    require(dir + '/utilities/link/link-underline.js')(),
-    require(dir + '/utilities/scrolling/smooth-scroll.js')(),
+    require(dir + "/utilities/accessibility/accessibility-hidden.js")(),
+    require(dir + "/utilities/accessibility/sr-only-text.js")(),
+    require(dir + "/utilities/link/link.js")(),
+    require(dir + "/utilities/link/link-fontweight.js")(),
+    require(dir + "/utilities/link/link-underline.js")(),
+    require(dir + "/utilities/scrolling/smooth-scroll.js")(),
 
     // 3rd Party Plugins;
-    require('@tailwindcss/aspect-ratio'),
-    require('@tailwindcss/forms'),
-    require('tailwindcss-debug-screens'),
-    require('tailwindcss-interaction-variants'),
-
+    require("@tailwindcss/aspect-ratio"),
+    require("@tailwindcss/forms"),
+    require("tailwindcss-debug-screens"),
+    require("tailwindcss-interaction-variants"),
   ],
 
   // The variants section lets you control which variants are generated for each
   // core utility plugin.
   // https://tailwindcss.com/docs/configuration/#variants
   variants: {
     extend: {
-      transform: ['group-hocus', 'motion-safe', 'motion-reduce'],
-      translate: ['group-hocus'],
-      rotate: ['group-hocus'],
-      margin: ['first', 'last'],
-      backgroundColor: ['group-focus-within', 'group-focus-visible', 'group-active', 'hocus', 'group-hocus'],
-      borderColor: ['group-focus-within', 'hocus', 'group-hocus'],
-      textColor: ['group-focus-within', 'group-focus-visible', 'group-active', 'hocus', 'group-hocus'],
-    }
-  }
-}
+      transform: ["group-hocus", "motion-safe", "motion-reduce"],
+      translate: ["group-hocus"],
+      rotate: ["group-hocus"],
+      margin: ["first", "last"],
+      backgroundColor: [
+        "group-focus-within",
+        "group-focus-visible",
+        "group-active",
+        "hocus",
+        "group-hocus",
+      ],
+      borderColor: ["group-focus-within", "hocus", "group-hocus"],
+      textColor: [
+        "group-focus-within",
+        "group-focus-visible",
+        "group-active",
+        "hocus",
+        "group-hocus",
+      ],
```

Comment:
```suggestion
      transform: ["motion-safe", "motion-reduce"],
      margin: ["first", "last"],
      backgroundColor: [
        "group-focus-within",
        "group-focus-visible",
        "group-active",
      ],
      borderColor: ["group-focus-within"],
      textColor: [
        "group-focus-within",
        "group-focus-visible",
        "group-active",
      ],
```
@sherakama I'm thinking that we can take out the "hocus" ones since they're just nice to have, and they're available for JIT mode already.